### PR TITLE
docs: use pkgdown BS5

### LINKED ISSUE #31: pkgdown website?
It would be great if a pkgdown website could be added.

### PR DESCRIPTION
**Pull request by maelle**
Fix #31 

- BS5
- low-hanging fruits: adding authors' URLs
- ignoring docs/
- making the duckdb-package manual page internal, as it brings nothing to the reference index.

I need to build the website locally to check things, then this will be ready for review.

### CODE REVIEW

**Comment by maelle** [comment]
This looks mostly good, apart from the giant logo. @krlmlr would it be worth for me to explore how to make the logo a PNG so it's not as huge in the pkgdown website?

**Comment by krlmlr** [comment]
Please do. Thanks!

File: .gitignore [hunk 1]
```diff
@@ -108,3 +108,4 @@ src/*.so
 /src/Makevars.duckdb
 /.ccache
 /src/symbols.rds
+docs
```
Comment (krlmlr) [inline .gitignore:111]: A little more precise:

```suggestion
/docs/
```

File: _pkgdown.yml [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
 
+template:
+  bootstrap: 5
+
+authors:
+  Hannes Mühleisen:
+    href: "http://hannes.muehleisen.org/"
+  Mark Raasveldt:
+    href: "https://mytherin.github.io/"
+
```
Comment (krlmlr) [inline _pkgdown.yml:11]: Add https://krlmlr.info/ for myself too?

Comment (maelle) [reply _pkgdown.yml:11]: You are not listed as an author (yet?)!

**Comment by maelle** [comment]
image


image

**Comment by maelle** [comment]
@krlmlr I ran `document()` to make the logo appear on the package-level manual page, which is how I realized it was created manually and with the deprecated docType, so I updated that as well. Created this way, the package-level manual page is itnernal so pkgdown won't add it in the index. This PR is now really ready.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**